driver-async/src/main/com/mongodb/async/client/internal/CommandMarker.java

```diff
@@ -79,13 +79,18 @@ public void onResult(final RawBsonDocument result, final Throwable t) {
                 }
             }
         };
-        executeCommand(databaseName, command, new SingleResultCallback<RawBsonDocument>() {
+        runCommand(databaseName, command, new SingleResultCallback<RawBsonDocument>() {
```

Comment:
This is simplified a bit since we're not longer "spawning if necessary" before executing the command (that's now happening in the constructor).